### PR TITLE
Next slice templates - AlternateGrid

### PR DESCRIPTION
**Pull request by hypervillain**
closes https://linear.app/prismic/issue/DT-1563/aauser-i-have-a-new-slice-template-available-alternate-grid

### CODE REVIEW

File: packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/sliceTemplatesComingSoon.ts [hunk 1]
```diff
@@ -1,39 +1,6 @@
...
-          docURL: "",
-          id: "Other",
-          name: "Other",
-          version: "sktwi1xtmkfgx8626",
-        },
-      ],
-    },
-    screenshots: {
-      default: {
-        url: "/alternateGrid.png",
```
Comment (xrutayisire) [inline packages/slice-machine/lib/builders/CustomTypeBuilder/SliceZone/sliceTemplatesComingSoon.ts:33]: Don't forget to remove the image also in public

**Comment by xrutayisire** [comment]
Was not possible to have mobile support as in the Storybook?

Screenshot 2023-09-08 at 16 32 33
Screenshot 2023-09-08 at 16 32 39

File: packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts [hunk 2]
```diff
@@ -0,0 +1,939 @@
...
+										: "es-alternate-grid__image--left"
+									}
+								\`}
+							/>
+						)}
+						<div className="es-alternate-grid__primary-content">
+							<div className="es-alternate-grid__primary-content__intro">
+								{isFilled.keyText(slice.primary.eyebrowHeadline) && (
+									<p className="es-alternate-grid__primary-content__intro__eyebrow">
+										{slice.primary.eyebrowHeadline}
```
Comment (xrutayisire) [inline packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts:492]: No need to use simple PrismicText

**Comment by xrutayisire** [comment]
Can you check the indentation to be like the CallToAction one:

For the inline style, you will see that it's a bit tricky to make it perfect after creation but it's working for CallToAction 🙂

Screenshot 2023-09-08 at 16 34 13
Screenshot 2023-09-08 at 16 35 20
Screenshot 2023-09-08 at 16 34 08

File: packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts (hunk 2 above)
Comment (hypervillain) [reply packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts:492]: I don't understand

Comment (xrutayisire) [reply packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts:492]: Sorry I wanted to add a question mark haha

Should we use PrismicText for these kind of things?

Comment (hypervillain) [reply packages/adapter-next/src/sliceTemplates/AlternateGrid/index.ts:492]: It depends if the designer considers that something else than a p tag should be used. In this case, I reused the first implementation from Sarah and Angelo

**Review by xrutayisire — APPROVED** [review]
LGTM

**Pull request merged**